src/base/image_reader.cc

```diff
@@ -181,15 +181,30 @@ ImageReader::Status ImageReader::Next(Camera* camera, Image* image,
       return Status::CAMERA_SINGLE_DIM_ERROR;
     }
 
+    //////////////////////////////////////////////////////////////////////////////
+    // Read camera model and check for consistency if it exists
+    //////////////////////////////////////////////////////////////////////////////
+    std::string camera_model = "Undefined";
+    bitmap->ExifCameraModel(&camera_model);
```

Comment:
That is a good suggestion. So far the model/make tags along with width and height have provided uniqueness for the data available to us. Including the focal length does make sense and if there are other tags that could be used I can include those as well.

Reply:
It would be great, if you could add focal length to the "unique hashing". A short documentation of the trade-offs of using "auto" could also be great. Thanks! Then I am happy to merge this.